astroid/nodes/node_ng.py

```diff
@@ -246,7 +246,7 @@ def accept(self, visitor):
     def get_children(self) -> Iterator[NodeNG]:
         """Get the child nodes below this node."""
         for field in self._astroid_fields:
-            attr = getattr(self, field)
+            attr = getattr(self, field, None)
```

Comment:
For which of the nodes is this necessary? `get_children` isn't only used in `__str__` so I'm worried people might depend on this returning an `AttributeError`.